front/src/components/Search/SearchBar.tsx

```diff
@@ -1,8 +1,100 @@
-const SearchBar = () => {
+import {
+  Accordion,
+  Button,
+  Container,
+  TextInput,
+  createStyles,
+  rem,
+} from '@mantine/core';
+import { IconSearch } from '@tabler/icons-react';
+import { useTranslation } from 'react-i18next';
+import CapacityRange from './Filters/CapacityRange';
+import GenreSelector from './Filters/GenreSelector';
+import ProfileType from './Filters/ProfileType';
+
+interface ISearchBarProps {
+  form: any;
+  onSubmit: (values: any) => void;
+}
+
+const useStyles = createStyles((theme) => ({
+  root: {
+    backgroundColor:
+      theme.colorScheme === 'dark'
+        ? theme.colors.dark[6]
+        : theme.colors.gray[0],
+    borderRadius: theme.radius.sm,
+  },
+
+  item: {
+    backgroundColor:
+      theme.colorScheme === 'dark' ? theme.colors.dark[6] : '#fff',
+    border: `${rem(1)} solid transparent`,
+    position: 'relative',
+    zIndex: 0,
+    transition: 'transform 150ms ease',
+
+    '&[data-active]': {
+      transform: 'scale(1.03)',
+      backgroundColor:
+        theme.colorScheme === 'dark' ? theme.colors.dark[7] : theme.white,
+      boxShadow: theme.shadows.md,
+      borderColor:
+        theme.colorScheme === 'dark'
+          ? theme.colors.dark[4]
+          : theme.colors.gray[2],
+      borderRadius: theme.radius.md,
+      zIndex: 1,
+    },
+  },
+
+  chevron: {
+    '&[data-rotate]': {
+      transform: 'rotate(-90deg)',
+    },
+  },
+}));
+
+const SearchBar: React.FC<ISearchBarProps> = ({ form, onSubmit }) => {
+  const { t } = useTranslation();
+  const { classes } = useStyles();
+
   return (
-    <div className="w-full flex justify-center">
-      <h1>Search Bar</h1>
-    </div>
+    <Container className="w-90 flex justify-center flex-col mt-4">
+      <form onSubmit={form.onSubmit((values: any) => onSubmit(values))}>
+        <Accordion
+          mx="auto"
+          variant="filled"
+          defaultValue="customization"
```

Comment:
you can remove this if you don't want the search accordion to open

Reply:
to open by default*

Reply:
if you do this, you can't have access to the search button, in french will call this "bien baisé" :upside_down_face: